.github/workflows/release_mgbench_bolt_client.yaml

```diff
@@ -0,0 +1,62 @@
+name: "Mgbench Bolt Client Publish Docker Image"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Mgbench bolt client version to publish on Dockerhub."
+        required: true
+      force_release:
+        type: boolean
+        required: false
+        default: false
+
+jobs:
+  mgbench_docker_publish:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_ORGANIZATION_NAME: memgraph
+      DOCKER_REPOSITORY_NAME: mgbench
```

Comment:
We should change the repository name to `mgbench bolt client ` Since the docker image contains just that, nothing else. This mgbench name is to  general for this. @brunos252

Reply:
One more thing, pulling image on Macbook yields following message: no matching manifest for linux/arm64/v8 in the manifest list entries.